src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -72,6 +78,14 @@ const visitClassDeclaration = (
     listenDecoratorsToStatic(diagnostics, decoratedMembers, filteredMethodsAndFields);
   }
 
+  // Handle static members that are initialized
+  const hasStaticMembersWithInit = classMembers.some(hasStaticInitializerInClass);
```

Comment:
I have a question as to whether this falls within the ambit of this transformer - as-named it's concerned with transforming properties decorated with 'stencil decorators' to static getters on the class.

Is it possible that the handling of `static` members should live in a separate transformer?

Reply:
Possibly - but since we're going to partially revert this once we upgrade to TS 5.1 (and remove this metadata), I'd prefer keeping it for now. LMK if today's 5.1 revelation changes your opinion from an hour ago 😆

Reply:
haha yeah, that makes sense I think